WooCommerce/Classes/ViewRelated/Orders/Order Details/Issue Refunds/RefundConfirmationViewController.swift

```diff
@@ -84,6 +88,14 @@ private extension RefundConfirmationViewController {
         }
         tableView.updateFooterHeight()
     }
+
+    /// Hides the keyboard by asking the view's first responder to resign on each main view tap.
+    ///
+    func configureKeyboardDismissal() {
+        let tap = UITapGestureRecognizer(target: view, action: #selector(UIView.endEditing))
+        tap.cancelsTouchesInView = false
+        view.addGestureRecognizer(tap)
+    }
```

Comment:
Changes on this file come from #3248 which was fixed on release/5.6 soon after code-freeze (no new beta yet)